README.md

```diff
@@ -1,3 +1,57 @@
-# rules_proto
+# Protobuf Rules for [Bazel](https://bazel.build)
 
-This repository contains Bazel rules for protocol buffers.
+This repository contains Starlark implementation of Protobuf rules in Bazel.
+
+For the list of Proto rules, see the Bazel
+[documentation](https://docs.bazel.build/versions/master/be/overview.html).
+
+## Getting Started
+
+There is no need to use rules from this repository just yet. If you want to use
+`rules_proto` anyway, add the following to your WORKSPACE file:
+
+```python
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rules_proto",
+    urls = ["https://github.com/bazelbuild/rules_proto/archive/TODO"],
+    sha256 = "TODO",
+)
+load("//proto:repositories.bzl", "proto_import_dependencies")
```

Comment:
There should be rules_proto_dependencies, and you can load both functions with one `load` (just pass multiple function names to load)